src/app/components/forms/checkbox-group/terra-checkbox-group.component.ts

```diff
@@ -0,0 +1,98 @@
+import {
+    Component,
+    forwardRef,
+    Input
+} from '@angular/core';
+import {
+    ControlValueAccessor,
+    NG_VALUE_ACCESSOR
+} from '@angular/forms';
+import { TerraMultiCheckBoxValueInterface } from '../../forms/multi-check-box/data/terra-multi-check-box-value.interface';
+
+@Component({
+    selector:  'terra-checkbox-group',
+    template:  require('./terra-checkbox-group.component.html'),
+    styles:    [require('./terra-checkbox-group.component.scss')],
+    providers: [
+        {
+            provide:     NG_VALUE_ACCESSOR,
+            useExisting: forwardRef(() => TerraCheckboxGroupComponent),
+            multi:       true
+        }
+    ]
+})
+export class TerraCheckboxGroupComponent implements ControlValueAccessor
+{
+    /**
+     * @description If true, the multi check box will be disabled. Default false.
+     * */
+    @Input()
+    public inputIsDisabled:boolean;
```

Comment:
wir wollen den prefix 'input' zukünftig weg lassen
```suggestion
    public isDisabled:boolean;
```